apps/docs/content/components/Section/index.mdx

```diff
@@ -0,0 +1,163 @@
+
+```js
+import {Section} from '@primer/react-brand'
+```
+
+## Examples
+
+### Default
+
+The default section wraps its content in a semantic section element and provides default paddings.
+
+```jsx live
+<Section>
+  <SectionIntro align="center">
+    <SectionIntro.Heading size="3">Default section</SectionIntro.Heading>
+    <SectionIntro.Description>
+      This content is wrapped in a Section component.
+    </SectionIntro.Description>
+  </SectionIntro>
+</Section>
+```
+
+### With indepenedent block paddings
+
+```jsx live
+<Section paddingBlockStart="none" paddingBlockEnd="spacious">
+  <SectionIntro align="center">
+    <SectionIntro.Heading size="3">Custom paddings</SectionIntro.Heading>
+    <SectionIntro.Description>
+      This section has a custom padding at the top and bottom.
+    </SectionIntro.Description>
+  </SectionIntro>
+</Section>
+```
+
+### With predefined background color
+
+```jsx live
+<Section backgroundColor="inset">
+  <SectionIntro align="center">
+    <SectionIntro.Heading size="3">Predefined background</SectionIntro.Heading>
+    <SectionIntro.Description>
+      This section has a predefined background color.
+    </SectionIntro.Description>
+  </SectionIntro>
+</Section>
+```
+
+### With custom background color
+
+```jsx live
+<Section backgroundColor="#FFF8F8">
+  <SectionIntro align="center">
+    <SectionIntro.Heading size="3">Custom background</SectionIntro.Heading>
+    <SectionIntro.Description>
+      This section uses a custom background color.
+    </SectionIntro.Description>
+  </SectionIntro>
+</Section>
+```
+
+### With background image
+
+```jsx live
+<Section
+  backgroundImageSrc="https://github.com/user-attachments/assets/068634a3-926d-49b8-b949-0b1e2a204651"
+  backgroundImageSize="cover"
+  backgroundImagePosition="top center"
+>
+  <ThemeProvider colorMode="dark">
+    <Grid>
+      <Grid.Column>
+        <SectionIntro align="center">
+          <SectionIntro.Heading size="3">Section</SectionIntro.Heading>
+          <SectionIntro.Description>
+            This section has a custom background image.
+          </SectionIntro.Description>
+        </SectionIntro>
+      </Grid.Column>
+    </Grid>
+  </ThemeProvider>
+</Section>
+```
+
+### With multiple background images
+
+```jsx live
+<ThemeProvider colorMode="dark">
+  <Section
+    paddingBlockStart="spacious"
+    paddingBlockEnd="spacious"
+    backgroundColor="default"
+    backgroundImageSrc={[
+      'https://github.com/user-attachments/assets/a7340dea-2076-42da-87a9-057f953beb83',
+      'https://github.com/user-attachments/assets/a28110fd-d019-41a4-8f80-b49ae8895708',
+    ]}
+    backgroundImagePosition={['bottom -1px center', 'center']}
+    backgroundImageSize={['auto', 'cover']}
+  >
+    <SectionIntro align="center">
+      <SectionIntro.Heading size="3" size="3">
+        Multiple background images
+      </SectionIntro.Heading>
+      <SectionIntro.Description>
+        This section uses multiple layered background images.
```

Comment:
Let's remove this description as it's not necessary and is currently causing some contrast issues that could get flagged down the line...

![Screenshot 2024-07-30 at 16 23 17](https://github.com/user-attachments/assets/fb3ad6e2-9143-4a68-ba3b-273f141da1b1)